jigsaw.json

```diff
@@ -31,7 +31,7 @@
     "persist": [
         "Jigsaw"
     ],
-    "checkver": "<!-- news section-->\\s+<!-- Jigsaw release -->\\s+<ul>\\s+<li><b>Released \\w{3,9} \\d+\\w+, \\d{4}: Jigsaw (?<version>((\\d+\\.)+\\d+))",
+    "checkver": "<!-- news section-->\\s+<!-- Jigsaw release -->\\s+<ul>\\s+<li><b>[ ,:\\w\\d]+(?<version>((\\d+\\.)+\\d+))",
```

Comment:
Keep it simple and use [`Released[^:]+:\s+Jigsaw\s+([\d.]+)`](https://regex101.com/r/dKrZkh/1) or even simpler `jigsaw_([\d.]+).zip`